rtc/jwt-authentication.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: RTC JWT Authentication
+title_nav: JWT Authentication
+description: Guide on how to setup JWT Authentication for RTC
+keywords: jwt authentication
+---
+
+{% assign pluginname = "RTC" %}
+{% assign plugincode = "rtc" %}
+## Introduction
+
+RTC requires you to setup JSON Web Token (JWT) authentication. This is to ensure that only your authenticated users will be able to access and collaborate on documents.
+
+JWT is a standard authorization solution for web services and is documented in more detail at the [https://jwt.io/](https://jwt.io/) website. The guide aims to show how to setup JWT authentication for RTC.
+
+{% include auth/private-public-key-pairs-for-tiny-cloud-services.md %}
+
+{% include auth/jwt-provider-endpoint-url.md %}
+
+## JWT requirements
+
+{% include auth/jwt-supported-algorithms.md %}
+
+### Claims
+
+These are like options/data you can send with the JWT token.
+
+| Data | Optional or required | Description |
+|---|:---:|---|
+| `sub` | required | The unique user ID (i.e. if `sub` is the same for two clients, you should trust them as if they're the same user). |
+| `exp` | required | The timestamp when the token expires. |
+
+{% include auth/jwt-endpoint-setup-procedure.md %}
+
+## Need help?
+
+We recommend reading up and trying to understand how JWT works; you need some necessary skills to implement RTC. This can be tricky, so if you need some help contact our support.
```

Comment:
```suggestion
Tiny recommends looking into how JWT works; some knowledge about JWT is necessary to implement RTC. This can be tricky, so if you need some help contact our support.
```
maybe?